rclcpp/test/rclcpp/test_duration.cpp

```diff
@@ -237,3 +237,16 @@ TEST_F(TestDuration, conversions) {
     EXPECT_EQ(chrono_duration.count(), -ONE_AND_HALF_SEC_IN_NS);
   }
 }
+
+TEST_F(TestDuration, test_some_constructors) {
```

Comment:
Is it possible to add coverage for the exceptions thrown during the operators `-` (which calls `bounds_check_duration_difference`) and `*` (which just needs a check that multiplying by `inf` catches the appropriate exception)?

Reply:
I'll add them before merging! checking Windows CI first

Reply:
Ok, see 049ba01